### ISSUE TITLE
BP,RNN 和 LSTM暨《Supervised Sequence Labelling with Recurrent Neural Networks-2012》阅读笔记 | new Cairo()

### ISSUE THREAD

**Issue by cairoHy** [cairoHy/blog_comments#43]
https://cairohy.github.io/2016/11/23/machine-learning/BP-RNN-%E5%92%8C-LSTM%E6%9A%A8%E3%80%8ASupervised-Sequence-Labelling-with-Recurrent-Neural-Networks-2012%E3%80%8B%E9%98%85%E8%AF%BB%E7%AC%94%E8%AE%B0/ 

 一、BackPropagation $w_{jk}^l$：表示第$l-1$层第k个神经元到第$l$层第j个神经元的连接权重； $b_j^l$：表示第$l$层第j个神经元的偏置； $z_j^l$：表示第$l$层第j个神经元的带权输入； $a_j^l$：表示第$l$层第j个神经元的激活值； $\sigma$：表示一个激活函数(sigmoid,relu,tanh)；  z_j^l = \sum_kw_{
